INSTALL.rst

```diff
@@ -12,12 +12,12 @@ In order to install DFTB+, you need the following software components:
 
 * A C-compiler
 
-* The C-preprocessor cpp
-
 * GNU make (version >= 3.79.1)
 
 * LAPACK/BLAS libraries (or compatible equivalents)
 
+* Optionally: ScaLAPACK, if you build the MPI-parallelised version of the code
```

Comment:
Version 2.0.0 or later for the PDSYEVR routines

Reply:
Added.